recipes/make-sense/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "make-sense" %}
+{% set version = "1.6.0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/SkalskiP/make-sense/archive/{{ version }}-alpha.tar.gz
+  sha256: c61b2a5926c93da57694a42f29efd14cb99f1fb313be76269babe6efb78bbecb
+
+build:
+  number: 0
```

Comment:
I guess the built package doesn't have any platform dependence?
```suggestion
  number: 0
  noarch: generic
```

Reply:
thanks, just added that

Reply:
@chrisburr getting an overlinking error on linux with noarch

Reply:
Ah yes, the weird compiler dependence means you need:
```suggestion
  number: 0
  noarch: generic
  ignore_run_exports:
    - libstdc++
```

Reply:
@chrisburr still got the same error. tried `libstdcxx-ng` and `libgcc-ng` in ignore_run_exports as well but those did also not work

Reply:
@chrisburr seems to be fixed now with missing_dso_whitelist for `libgcc_s.so.1` and `libstdc++.so.6`